bikeshare-example-database.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Bikeshare Example Database
+summary: Install the Bikeshare Example Database
+---
+
+# Bikeshare Example Database
+
+Examples used in the TiDB manual use [System Data](https://www.capitalbikeshare.com/system-data) from 
+Capital Bikeshare, released under the [Capital Bikeshare Data License Agreement](https://www.capitalbikeshare.com/data-license-agreement).
+
+## Downloading all data files
+
+The system data is available [for download in .zip files](https://s3.amazonaws.com/capitalbikeshare-data/index.html) organized per year.  Downloading and extracting all files requires approximately 3GB of disk space.  To download all files for years 2010-2017 using a bash script:
+
+```
```

Comment:
Pls add `bash` for the code fencing. And also add corresponding code fencing for other code blocks in this PR.